conanfile.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class AbseilConan(ConanFile):
+    name = "abseil"
+    url = "https://github.com/abseil/abseil-cpp"
+    homepage = url
+    author = "Ashley Hedberg <ahedberg@google.com>"
```

Comment:
I'm pretty sure I didn't write this?

Reply:
`author` is a reference of who is responsible for the recipe. we could change of course, do you prefer use other name or team name?

Reply:
Let's put the Abseil mailing list (abseil-io@googlegroups.com) here instead.

Reply:
done